src/lib/snarky_js_bindings/lib/snarky_js_bindings_lib.ml

```diff
@@ -2231,6 +2231,7 @@ let pickles_compile (choices : pickles_rule_js Js.js_array Js.t)
     Pickles.compile_promise () ~choices
       ~public_input:(Input (public_input_typ public_input_size))
       ~auxiliary_typ:Typ.unit
+      ~override_wrap_domain:Pickles_base.Proofs_verified.N1
```

Comment:
Just for my education, how does this change work? Was `compile_promise` working off some default that is now changed to `N1`? Why N1? :thinking: :thought_balloon:

Reply:
Ye, it seems like the default was changed somewhere, which caused `ZkProgram` not to function correctly when we wanted to recursively verify proofs - now it seems to work fine again :D

Reply:
Did you test with 0, 1 and 2 max proofs verified?